tokio/src/fs/exists.rs

```diff
@@ -0,0 +1,27 @@
+use std::path::Path;
+
+/// Returns `true` if the path points at an existing entity.
+///
+/// This function will traverse symbolic links to query information about the
+/// destination file.
+///
+/// # Examples
+///
+/// ```rust,no_run
+/// # use tokio::fs;
+///
+/// # #[tokio::main]
```

Comment:
The example in `src/fs/metadata.rs` does not hide these lines. Maybe we should be consistent and show these lines?